workbench-core/accounts/src/errors/errors.test.ts

```diff
@@ -11,4 +12,10 @@ describe('custom error tests', () => {
 
     expect(isInvalidAwsAccountIdError(invalidAwsAccountIdError)).toBe(true);
   });
+
+  test('InvalidAccountError', () => {
+    const error = new InvalidAccountStateError();
+
+    expect(isInvalidAccountStateError(error)).toBe(true);
```

Comment:
nit: should also check that isValidAccountStateError can evaluate to false if the passed error is of a different type